.github/workflows/deploy.yml

```
@@ -0,0 +1,76 @@
+name: Deploy contracts
+
+on:
+  workflow_dispatch: # Manual trigger
+    inputs:
+      network:
+        description: 'Network'
+        required: true
+        type: choice
+        options:
+          - hardhat
+          - bellecour
+          - arbitrumSepolia
+          - arbitrum
+        default: 'hardhat'
+      whitelist_owner:
+        description: 'Whitelist contract owner address (optional - defaults to deployer)'
+        required: false
+        type: string
```

```suggestion
      whitelist_owner:
        description: 'Whitelist contract owner address'
        required: true
        type: string
```
The deployment should fail if the owner is not specified. The deployer is not supposed to be the owner.